File: ahnmikawhite.java
import java.util.concurrent.atomic.*;
public class ahnmikawhite{
	static AtomicInteger flag = new AtomicInteger(0);
	static int count=0;
	public static void main(String[] args){
		Runtime.getRuntime().addShutdownHook(new Thread(){
			@Override
			public void run(){
				flag.set(1);
				System.out.println(count+" 人のアンミカ");
			}
		});
		while(flag.get()==0){
			System.out.println("白って"+ (int)(Math.random()*(1001-100)+100) +"色あんねん");
			count++;
		}
	}
}

File: AhnmikaWhite.cs
using System;

class AhnmikaWhite
{
    static void Main()
    {
        var rnd = new Random();
        var flag = true;
        UInt64 count = 0;

        Console.CancelKeyPress += (object sender, ConsoleCancelEventArgs e) => {flag = false; e.Cancel = true;};
        
        while (flag)
        {
            var colors = rnd.Next(100, 1001);
            Console.WriteLine("白って{0}色あんねん", colors);
            count++;
        }
        Console.WriteLine("{0} 人のアンミカ", count);
    }
}
